File: src/test/samples/validSamples/validSample.py
#region FirstRegion
x = 42
#endregion

# region Second Region
class MyClass:
    #  region    InnerRegion    
    def method(self):
        pass
    #          endregion   ends InnerRegion   

    #region
    def method2(self):
        pass
    # endregion

# endregion

File: src/test/samples/validSamples/validSample.clj
;#region FirstRegion
(def x 42)
;;#endregion

;;;;   #region Second Region  
(defn my-class []
    ;   #region    InnerRegion   
    (defn my-method [] nil)
    ;;   #endregion    ends InnerRegion  

    ;;   #region  
    (defn my-method2 [] nil)
    ;;#endregion  
)
;; #endregion

